test/integration/crud/misc_cursors.test.js

```diff
@@ -2994,40 +2327,30 @@ describe('Cursor', function () {
       }
 
       const configuration = this.configuration;
-      client.connect((err, client) => {
-        expect(err).to.not.exist;
-        this.defer(() => client.close());
+      await client.connect();
```

Comment:
doesn't this test duplicate the above test?

Reply:
Yeah, this only differs in a single extra `.map(...)` call. Removing the test.